src/containers/App.jsx

```diff
@@ -3,27 +3,49 @@ import React from 'react'
 const myStorage = window.localStorage
 
 class App extends React.Component {
-    addLocation() {
+    componentDidMount() {
+        const initData = myStorage.getItem('initialData')
+        if (!initData) return null
+        const { lat, lon } = JSON.parse(initData)
+        const pos = (`${lat},${lon}`).split('.').join('-')
+        this.props.history.push(`/dashboard/@${pos}`)
+    }
+
+    addLocation = () => {
         navigator.geolocation.getCurrentPosition(data => {
             const position = { lat: data.coords.latitude, lon: data.coords.longitude }
             myStorage.setItem('initialData', JSON.stringify(position))
-            const pos = (`${position.lat},${position.lon}`).split('.').join('-')
-            this.props.history.push(`/dashboard/@${pos}`)
+            this.goToDepartureBoard(position)
         })
     }
 
-    componentDidMount() {
-        const initData = myStorage.getItem('initialData')
-        if (!initData) return this.addLocation()
-        const { lat, lon } = JSON.parse(initData)
-        if (!lat || !lon) return this.addLocation()
-        const pos = (`${lat},${lon}`).split('.').join('-')
+    goToDepartureBoard(position) {
+        const pos = (`${position.lat},${position.lon}`).split('.').join('-')
         this.props.history.push(`/dashboard/@${pos}`)
     }
 
+    handleLatlongSubmit = (event) => {
+        const latlong = { lat: event.target.lat.value, long: event.target.long.value }
+        this.goToDepartureBoard(latlong)
+        event.preventDefault()
+    }
+
+
     render() {
         return (
-            <h1>render</h1>
+            <div>
+                <h1>Velkommen til tavla.en-tur.no</h1>
+                <p>For at vi skal lage en tavle til deg trenger vi at du godtar at vi kan bruke din posisjon...</p>
+                <button onClick={this.addLocation}>Godta bruk av posisjon</button>
+                <p>...eller fyller inn relevante koordinater</p>
+                <form onSubmit={this.handleLatlongSubmit}>
+                    <label>Breddegrad </label>
```

Comment:
labels bør ha `for="something"` og input bør ha `id="something"` for å knytte label til input

Typ
```
<label for="longitude">Lengdegrad </label>
<input id="longitude" type="text" name="long" />
```